### PR TITLE
Adds Vegetative Poultice druid spell to summon magical bandages

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Content "Adds a Magiclysm druid healing spell to stop bleeding"

#### Purpose of change

Druids need more healing spells since they were probably the main healers in pre-Industrial times, so this adds a spell that turns bark and leaves into bandages. The bandages are more effective than makeshift bandages but less effective than modern bandages in keeping with Magiclysm's "modern technology is generally better" ethos, but have the major advantage that they have a very low applying time since you just slap the result on the wound rather than actually bind it.

#### Describe the solution

As above, add a spell to summon bandages

#### Describe alternatives you've considered

Use this spell in my personal game and don't add it to main.

#### Testing

I've been using it for a while and it works.

### CODE REVIEW

**Review by GuardianDll — APPROVED** [review]
Looks good for me

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -1238,5 +1238,24 @@
...
+    "looks_like": "leaves",
+    "description": "A mixture of bark, moss, and leaves, glowing faintly with verdant energy.",
+    "weight": "70 g",
+    "volume": "250 ml",
+    "price": 1,
+    "price_postapoc": 1,
+    "material": [ "wood" ],
+    "symbol": "!",
+    "color": "green",
+    "charges": 3,
```
Comment (GuardianDll) [inline data/mods/Magiclysm/items/ethereal_items.json:1256]: Does this means it spawn 3 bandages at one cast?

Comment (Standing-Storm) [reply data/mods/Magiclysm/items/ethereal_items.json:1256]: No, it only spawns one and each bandage only has one use. When I wrote this spell a couple months ago, every bandage had 3 charges (while still only having one use), so I assumed it was some weird legacy code thing I had to include. Bandages no longer have three charges, though, and testing shows it works fine without it, so I'll take it out.

**Comment by Standing-Storm** [comment]
Hmm, I'm not sure why changing that one line causes the Basic Build test to fail now

**Comment by GuardianDll** [comment]
```

(~[slow] ~[.],starting_items)=> ../tests/vehicle_part_test.cpp:193: FAILED:
(~[slow] ~[.],starting_items)=>   REQUIRE( character.get_wielded_item()->type->get_id() == recipe.result() )
(~[slow] ~[.],starting_items)=> with expansion:
Error: (~[slow] ~[.],starting_items)=>   string_id( "craft" )
(~[slow] ~[.],starting_items)=>   ==
(~[slow] ~[.],starting_items)=>   string_id( "oatmeal_cooked" )
(~[slow] ~[.],starting_items)=> 

```
i believe its not your fault

**Comment by Standing-Storm** [comment]
This should be good to go now, I think

**Pull request merged**